src/parser/mod.rs

```diff
@@ -4139,6 +4160,167 @@ impl<'a> Parser<'a> {
         })
     }
 
+    pub fn parse_create_trigger(&mut self, or_replace: bool) -> Result<Statement, ParserError> {
+        if dialect_of!(self is PostgreSqlDialect) {
+            let name = self.parse_object_name(false)?;
+            let period = self.parse_trigger_period()?;
+
+            let event = self.parse_keyword_separated(Keyword::OR, Parser::parse_trigger_event)?;
+            let table_name = if self.parse_keyword(Keyword::ON) {
+                self.parse_object_name(false)?
+            } else {
+                return self.expected("keyword `ON`", self.peek_token());
+            };
+
+            let mut referencing = vec![];
+            if self.parse_keyword(Keyword::REFERENCING) {
+                while let Some(refer) = self.parse_trigger_referencing()? {
+                    referencing.push(refer);
+                }
+            }
+
+            let (for_each, include_each) = if self.parse_keywords(&[Keyword::FOR]) {
+                let include_each = self.parse_keyword(Keyword::EACH);
+                (
+                    match self.parse_one_of_keywords(&[Keyword::ROW, Keyword::STATEMENT]) {
```

Comment:
we can use [self.expect_one_of_keywords](https://github.com/sqlparser-rs/sqlparser-rs/blob/main/src/parser/mod.rs#L3303)

Reply:
Done!